renderdoc/driver/vulkan/vk_pixelhistory.cpp

```diff
@@ -818,8 +820,12 @@ struct VulkanPixelHistoryCallback : public VulkanDrawcallCallback
 
     for(uint32_t i = 0; i < fbInfo.attachments.size(); i++)
     {
-      atts[i] = m_pDriver->GetResourceManager()->GetCurrentHandle<VkImageView>(
-          fbInfo.attachments[i].createdView);
+      ResourceId imageView;
+      if(fbInfo.imageless)
+        imageView = pipestate.GetFramebufferAttachments()[i];
```

Comment:
`GetFramebufferAttachments()` is set up so that it either returns the dynamic attachments, or for old style framebuffers it returns the one from the creation info. It's worth double checking, but I believe it should be safe just to call always.

Reply:
Updated, thanks! Tested on 2 simple apps: imageless and normal framebuffer.